src/page/managers/LoginManager.ts

```diff
@@ -39,33 +43,36 @@ export default class LoginManager {
     // avoid duplicate identity requests, this is needed if dev calls init and login in quick succession e.g.
     // e.g. OneSignalDeferred.push(OneSignal) => OneSignal.init({...})); OneSignalDeferred.push(OneSignal) => OneSignal.login('some-external-id'));
     identityModel.setProperty(
-      'external_id',
+      IdentityConstants.EXTERNAL_ID,
       externalId,
       ModelChangeTags.HYDRATE,
     );
     const newIdentityOneSignalId = identityModel.onesignalId;
-
     const appId = MainHelper.getAppId();
 
-    const pushOp = await OneSignal.coreDirector.getPushSubscriptionModel();
-    if (pushOp) {
-      OneSignal.coreDirector.operationRepo.enqueue(
-        new TransferSubscriptionOperation(
+    const promises: Promise<void>[] = [
+      OneSignal.coreDirector.getPushSubscriptionModel().then((pushOp) => {
+        if (pushOp) {
+          OneSignal.coreDirector.operationRepo.enqueue(
+            new TransferSubscriptionOperation(
+              appId,
+              newIdentityOneSignalId,
+              pushOp.id,
+            ),
+          );
+        }
+      }),
+      OneSignal.coreDirector.operationRepo.enqueueAndWait(
+        new LoginUserOperation(
           appId,
           newIdentityOneSignalId,
-          pushOp.id,
+          externalId,
+          !currentExternalId ? currentOneSignalId : undefined,
         ),
-      );
-    }
-
-    await OneSignal.coreDirector.operationRepo.enqueueAndWait(
-      new LoginUserOperation(
-        appId,
-        newIdentityOneSignalId,
-        externalId,
-        !currentExternalId ? currentOneSignalId : undefined,
       ),
-    );
+    ];
+
+    await Promise.all(promises);
```

Comment:
Just double checking, do we not care about the order in which `TransferSubscriptionOperation` and `LoginUserOperation` are run?